server/middleware/workflow-client.js

```diff
@@ -129,6 +129,7 @@ function enumTransform(item) {
     'continue_as_new_initiator',
     'event_type_timer',
     'timeout_type',
+    'archival_status',
```

Comment:
Can you add of them here? At least `retry_status`.

Reply:
I've added retry_status as part of current PR. I will add the rest in the near time in a separate PR